tests/docs/smoke-all/2024/10/29/test-shortcode-json-filter-insertion/insert_shortcode.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+import json
+import sys
+
+inp = sys.stdin.read()
+doc = json.loads(inp)
+
+#######
+# p_* are a bad version of a micro-panflute
+
+def p_string(string):
+  return {
+    "t": "String",
```

Comment:
Should this be `"t": "Str"`? It seems that this function isn't used.

Reply:
_It's been zero days since a GitHub Copilot accident._